src/pages/FlagCommentPage.js

```diff
@@ -120,18 +124,19 @@ function FlagCommentPage(props) {
 
         // Handle threads if needed
         if (reportAction === undefined || reportAction.reportActionID === undefined) {
-            reportAction = ReportActionsUtils.getParentReportAction(props.report);
+            reportAction = props.parentReportActions[`${props.report.parentReportActionID}`] || {};
```

Comment:
I think we can remove the conversion to string since `parentReportActionID` should already be a [string](https://github.com/Expensify/App/blob/b9c2514294d64e969f1eb07a4b75989340ffd06c/src/types/onyx/Report.ts#L112), no?
```suggestion
            reportAction = props.parentReportActions[props.report.parentReportActionID] || {};
```

Reply:
Oh, true. The template string isn't necessary anymore.